.swiftlint.yml

```diff
@@ -37,7 +40,7 @@ custom_rules:
     included: ".*\\.swift" # regex that defines paths to include during linting. optional.
     excluded: ".*Test(s)?\\.swift" # regex that defines paths to exclude during linting. optional
     name: "Commented out code" # rule name. optional.
-    regex: "^\\/\\/\\s*(@|\\.?([a-z]|(\\})))" # matching pattern
+    regex: "^\\s*(\/\/(?!\\s*swiftlint:).*|\/\\*[\\s\\S]*?\\*\/)" # matching pattern
```

Comment:
Had to modify it as it was triggering on lines that start with `// swiftlint:` that we use to disable/enable rules.